gst/nnstreamer/tensor_merge/gsttensormerge.c

```diff
@@ -591,23 +581,22 @@ gst_tensor_merge_generate_mem (GstTensorMerge * tensor_merge,
             memcpy (outptr, inptr, s);
             outptr += s;
           }
-
-          ret = TRUE;
-        }
           break;
+        }
         default:
-          ret = FALSE;
+          ret = GST_FLOW_ERROR;
       }
-    }
       break;
+    }
     default:
-      ret = FALSE;
+      ret = GST_FLOW_ERROR;
   }
 
+  gst_memory_unmap (outMem, &outInfo);
   gst_buffer_append_memory (tensor_buf, outMem);
   gst_buffer_copy_into (tensor_buf, tensors_buf, GST_BUFFER_COPY_TIMESTAMPS, 0,
       -1);
-  gst_memory_unmap (outMem, &outInfo);
+
```

Comment:
Since below is error hadler. How about add `return ret` (or change label name below) here as well?

Reply:
input-mem should be unmapped, so below lines are necessary here :)